doc/dev.md

```diff
@@ -202,6 +202,52 @@ cmake -S . -B build-x64-ios -DVCPKG_TARGET_TRIPLET=x64-ios -GXcode -DWITH_VCPKG=
 cmake --build build-x64-ios
 ```
 
+
+## iOS application (ARM-64 processors architecture)
+
+```sh
+# Firstly, some compilation dependencies need to be installed
+brew install cmake flex bison python pkg-config autoconf automake libtool
+
+# Secondly, Xcode must be installed through the AppStore, then configured (Sometimes commands like those are needed to correctly set-up Xcode)
+xcode-select --install
+sudo xcode-select --switch /Library/Developer/CommandLineTools
+sudo xcode-select -s /Applications/Xcode.app/Contents/Developer
+
+# Then, of course, Qt is needed (Could be installed with 'aqtinstall' or with the official tools)
+pip3 install aqtinstall && aqt install-qt mac ios 5.15.2 -O qt5 -m qtcharts # Qt
+
+# Then, really important, some variables need to be exported in the shell
+
+export PATH="$(brew --prefix flex)/bin:$PATH" && export PATH="$(brew --prefix bison)/bin:$PATH" # For QGis
+
+export Qt5_DIR=$HOME/Documents/qt5/5.15.2/ios/ # Seems that a full path might be needed for dependencies based on Qt like Qca
+
+# The Cmake configuration command should look like something like this one (The architecture parameter seems to be really important)
+
+cmake -S . -B build-arm64-ios \
+	-DCMAKE_PREFIX_PATH=$HOME/Documents/qt5/5.15.2/ios/lib/cmake/Qt5 \
+	-DCMAKE_FIND_ROOT_PATH=$HOME/Documents/qt5/5.15.2/ios/ \
+	-DSYSTEM_QT=ON \
+	-DWITH_QT5=1 \
+	-DVCPKG_TARGET_TRIPLET=arm64-ios \
+	-DWITH_VCPKG=ON \
+	-DVCPKG_BUILD_TYPE=release \
+	-DCMAKE_SYSTEM_NAME=iOS \
+	-DCMAKE_OSX_SYSROOT=iphoneos \
+	-DCMAKE_OSX_ARCHITECTURES=arm64 \
+	-GXcode
+
+# The "build/CMakeFiles/[version]/CMakeSystem.cmake" file may need to be modified.
+# The "CMAKE_SYSTEM_PROCESSOR" variable must be set to "aarch64" manually (only if the cmake configuration fails).
```

Comment:
I wish we could find a better way and avoid this manual patching requirement, but for now let's document it as done here, thanks!